awx/ui_next/src/screens/Template/shared/WorkflowJobTemplateForm.jsx

```diff
@@ -0,0 +1,454 @@
+                  onChange={value => {
+                    form.setFieldValue('inventory', value?.id || null);
+                    form.setFieldValue(
+                      'organizationId',
+                      value?.organization || null
+                    );
+                    setInventory(value);
+                  }}
+                  error={form.errors.inventory}
+                />
+              )}
+            </Field>
+            <FormGroup
+              fieldId="wfjt-limit"
+              name="limit"
+              label={i18n._(t`Limit`)}
+            >
+              <FieldTooltip
+                content={i18n._(
+                  t`Provide a host pattern to further constrain the list of hosts that will be managed or affected by the workflow. This limit is applied to all job template nodes that prompt for a limit. Refer to Ansible documentation for more information and examples on patterns.`
+                )}
+              />
+              <FormField type="text" name="limit" id="wfjt-limit" label="" />
+            </FormGroup>
+            <FormGroup
+              fieldId="wfjt-scmBranch"
+              id="wfjt-scmBranch"
+              label={i18n._(t`SCM Branch`)}
+              name="scmBranch"
+            >
+              <FieldTooltip
+                content={i18n._(
+                  t`Select a branch for the workflow. This branch is applied to all job template nodes that prompt for a branch.`
+                )}
+              />
+              <FormField
+                type="text"
+                id="wfjt-scmBranch"
+                label=""
+                name="scmBranch"
+              />
+            </FormGroup>
+          </FormRow>
+
+          <FormRow>
+            <Field name="labels">
+              {({ form, field }) => (
+                <FormGroup
+                  label={i18n._(t`Labels`)}
+                  name="wfjt-labels"
+                  fieldId="wfjt-labels"
+                >
+                  <FieldTooltip
+                    content={i18n._(t`Optional labels that describe this job template,
+                    such as 'dev' or 'test'. Labels can be used to group and filter
+                    job templates and completed jobs.`)}
+                  />
+                  <LabelSelect
+                    value={field.value}
+                    onChange={labels => form.setFieldValue('labels', labels)}
+                    onError={() => setContentError()}
+                  />
+                </FormGroup>
+              )}
+            </Field>
+          </FormRow>
+          <FormRow>
+            <VariablesField
+              id="wfjt-variables"
+              name="variables"
+              label={i18n._(t`Variables`)}
+            />
+          </FormRow>
+          <GridFormGroup
+            fieldId="options"
+            isInline
+            label={i18n._(t`Options`)}
+            css="margin-top: 20px"
+          >
+            <Checkbox
```

Comment:
I'd recommend taking a look at the `CheckboxField` component.  I think it shorten up this options code a bit.  We started using it in the job template form for the options section. https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx#L525